scripts/microbe_editor/microbe_editor_hud.lua

```diff
@@ -266,12 +266,15 @@ function MicrobeEditorHudSystem:nameClicked()
 end
 
 function MicrobeEditorHudSystem:updateMicrobeName()
+    --set genus_picked to false so it knows to update the name properly when microbe_replacement runs
+    global_Genus_Picked = false;

```

Comment:
Your variables name do not respect the naming convention. They should be camelCase, with no underscore (except global_).